src/openlifu/db/session.py

```diff
@@ -121,6 +140,11 @@ def from_dict(d:Dict):
             raise ValueError("Sessions no longer recognize a volume attribute -- it is now volume_id.")
         if 'array_transform' in d:
             d['array_transform'] = ArrayTransform(np.array(d['array_transform']['matrix']), d['array_transform']['units'])
+        if 'transducer_tracking_results' in d:
+            d['transducer_tracking_results'] = [TransducerTrackingResult(t['photoscan_id'],
+                                                                         ArrayTransform(np.array(t['transducer_to_photoscan_transform']['matrix']),t['transducer_to_photoscan_transform']['units']),
+                                                                         ArrayTransform(np.array(t['photoscan_to_volume_transform']['matrix']), t['photoscan_to_volume_transform']['units']),
+                                                                         t['transducer_tracking_approved']) for t in d['transducer_tracking_results']]
```

Comment:
More readable indentation:

```suggestion
        if 'transducer_tracking_results' in d:
            d['transducer_tracking_results'] = [
                TransducerTrackingResult(
                    t['photoscan_id'],
                    ArrayTransform(np.array(t['transducer_to_photoscan_transform']['matrix']),t['transducer_to_photoscan_transform']['units']),
                    ArrayTransform(np.array(t['photoscan_to_volume_transform']['matrix']), t['photoscan_to_volume_transform']['units']),
                    t['transducer_tracking_approved']
                )
                for t in d['transducer_tracking_results']
            ]
```